docker-compose.yml

```diff
@@ -1,173 +1,202 @@
-    ports:
-      - 3001:3000
-    working_dir: /opt/app
-    links:
-      - mongodb
-    depends_on:
-      - mongodb
-    volumes:
-      - ./packages/common:/opt/app/node_modules/@pins/common
-      - ./packages/document-service-api:/opt/app
-      - ./tmp/documents-upload:/tmp/upload
-    command: npm run start:dev
-
-  document-service-uploader:
-    <<: *document-service
-    ports: []
-    links:
-      - blob-storage
-      - mongodb
-    depends_on:
-      - blob-storage
-      - mongodb
-    command: node ./src/tasks/uploadFilesToBlobStorage
-
-  forms-web-app:
-    image: node:14-alpine
-    environment:
-      SERVER_LIMITED_ROUTING_ENABLED: ${SERVER_LIMITED_ROUTING_ENABLED}
-      APPEALS_SERVICE_API_URL: http://appeals-service-api:3000
-      DOCUMENTS_SERVICE_API_URL: http://document-service-api:3000
-      FILE_UPLOAD_DEBUG: "false"
-      FILE_UPLOAD_MAX_FILE_SIZE_BYTES: 50000000
-      FILE_UPLOAD_USE_TEMP_FILES: "true"
-      FILE_UPLOAD_TMP_PATH: "/tmp"
-      SESSION_MONGODB_URL: mongodb://mongodb:27017/fwa-sessions
-      SESSION_KEY: some_secure_key_goes_here
-      USE_SECURE_SESSION_COOKIES: "false"
-    ports:
-      - 9000:3000
-    working_dir: /opt/app
-    links:
-      - appeals-service-api
-      - document-service-api
-      - mongodb
-    depends_on:
-      - appeals-service-api
-      - document-service-api
-      - mongodb
-    volumes:
-      - ./packages/common:/opt/app/node_modules/@pins/common
-      - ./packages/forms-web-app:/opt/app
-    command: npm run start:dev
-
-  # Populate the database with data - one instance per service
-  appeals-service-api-data:
-    build:
-      context: ./data
-    links:
-      - mongodb
-    depends_on:
-      - mongodb
-    volumes:
-      - ./data:/opt/app
-    environment:
-      SOURCE_DIR: appeals-service-api
-      MONGODB_URL: mongodb://mongodb:27017/appeals-service-api
-    restart: on-failure
-    command: npm start
-
-  # Mocked services
-  mock-horizon:
-    # @todo generate from Swagger Docs
-    build:
-      context: ./mocked-services/horizon
-      dockerfile: ../Dockerfile
-    ports:
-      - 5000:3000
-
-  mock-notify:
-    build:
-      context: ./mocked-services/notify
-      dockerfile: ../Dockerfile
-    ports:
-      - 5001:3000
-
-  mock-os-places:
-    build:
-      context: ./mocked-services/os-places
-      dockerfile: ../Dockerfile
-    ports:
-      - 5002:3000
-
-  docs-horizon:
-    image: swaggerapi/swagger-ui
-    environment:
-      SWAGGER_JSON: /app/swagger.yaml
-    depends_on:
-      - mock-horizon
-    volumes:
-      - ./docs/swagger/horizon/create_case_openapi.yaml:/app/swagger.yaml
-    ports:
-      - 7000:8080
-
-  # Third-party services
-  blob-storage:
-    image: mcr.microsoft.com/azure-storage/azurite
-    ports:
-      - 4002:10000
-      - 4003:10001
-    volumes:
-      - ./tmp/blob-storage:/data
-
-  mongodb:
-    image: mongo:3.6.0
-    ports:
-      - 4000:27017
+# Port ranges (by convention):

```

Comment:
I'm confused why this is showing the whole file in red/green - I'll figure it out, but just work out why